### PR TITLE
[TS] LPS-162394

### PR DESCRIPTION
**Pull request by holatuwol**
Update to previous pull request https://github.com/liferay-frontend/liferay-portal/pull/2700 which should only affect the "replace" certificate command.

Also switched to `parent.window` instead of `Liferay.Util.getOpener()` due to a new React error that occurs in `frontend-js-state-web` that occurs when `aui:script` is present.

## Steps to reproduce

1. Go to [Control Panel -> SAML Admin]
1. Add any Entity ID
1. Create Certificate
1. Add Common Name
1. Add Key Password
1. Save

## Solution notes

https://liferay.slack.com/archives/CNBG06JS3/p1649760991802989

Based on testing, the issue was introduced when we switched to the v4 loader.

* 7.0 - not reproducible
* 7.1 - reproducible if loader v4 is enabled in system settings
* 7.2 - reproducible
* 7.3 - reproducible
* 7.4 - reproducible

### CODE REVIEW

**Review by markocikos — COMMENTED** [review]
@holatuwol please see comment inline.

File: modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/portlet/action/UpdateCertificateMVCActionCommand.java [hunk 1]
```diff
@@ -283,9 +284,12 @@ private void _importCertificate(ActionRequest actionRequest, User user)
 			SamlWebKeys.SAML_X509_CERTIFICATE, x509Certificate);
 	}
 
-	private void _replaceCertificate(ActionRequest actionRequest)
+	private void _replaceCertificate(
+			ActionRequest actionRequest, ActionResponse actionResponse)
 		throws Exception {
 
+		hideDefaultSuccessMessage(actionRequest);
+
```
[the remark this answers is not in the document]
Comment (holatuwol) [reply modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/portlet/action/UpdateCertificateMVCActionCommand.java:292]: @markocikos For the specific bug mentioned in the ticket, `hideDefaultSuccessMessage` works. However, once that issue is resolved, I got a new error for `frontend-js-state-web` which only went away when I applied the additional changes.

[the remark this answers is not in the document]
Comment (markocikos) [reply modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/portlet/action/UpdateCertificateMVCActionCommand.java:292]: What are the steps to this `frontend-js-state-web` error? I don't see any error in the steps of the ticket.

[the remark this answers is not in the document]
Comment (holatuwol) [reply modules/dxp/apps/saml/saml-web/src/main/java/com/liferay/saml/web/internal/portlet/action/UpdateCertificateMVCActionCommand.java:292]: @markocikos Same steps as the error reported in the ticket, just a different error.

**Review by markocikos — APPROVED** [review]
@holatuwol Sorry for the extremely long review delay :disappointed: 

It looks like this PR fixes even more serious bug now, where certificate creation completely breaks. I'll trust you on necessity of other changes, although I cannot replicate them. Maybe it's something weird on my site. What I can say is additional changes don't hurt for sure. Forwarding away...

**Comment by markocikos** [comment]
ci:forward

**Pull request closed without merging**